community/working-groups/cross-foundation-alignment/README.md

```diff
@@ -0,0 +1,27 @@
+# Cross-Foundation Alignment Working Group
+
+## Goals
+
+- Create a bridge for knowledge sharing between the STAG and other bodies within the Linux Foundation
+- Seek opportunities to mitigate duplication of security efforts between bodies within the Linux Foundation
+- Ensure hygiene recommendations for CNCF projects align closely with corresponding OpenSSF recommendations
+
+## Scope
+
+- Coordinate discussion between contributors from STAG and OpenSSF
+- Highlight opportunities for the STAG to contribute to codebases, standards, and publications that may benefit the CNCF and/or STAG goals
+
+## Deliverables
+
+1. Aid in the creation of a universal open source project security baseline.
+
+## Meeting Information
+
+- **Meeting:** Every 2 weeks on Tuesday at 2:00 PM Pacific Time (US and Canada) ([Calendar Invite](https://zoom.us/meeting/tJUtduGoqz4qGddkUvgs3jVjzUEY6Y8MEcT6/ics?icsToken=98tyKuCprjoiGtGQsBqERowcAoj4WfTwmCVfjadZlyrzBDMAaDX8LNdnC-RGSPX1))
+- **Meeting Notes:** [Google Docs](https://docs.google.com/document/d/1sa_dBQifM8Fbp1tmNEkdoZKYXMw4pCPv_TcGBy6M4O0/)
+
+## Contact
+
+- **Lead:** Eddie Knight
+- **STAG Rep:** Michael Lieberman
+- **Slack Channel:** [Link](https://cloud-native.slack.com/archives/C06B26A12AF)
```

Comment:
The slack URL for the (currently named, probably will change?) #tag-security-baseline-wg channel is https://app.slack.com/client/T08PSQ7BQ/C07EX4ZU15Y

Reply:
```suggestion
- **Slack Channel:** [Link](https://app.slack.com/client/T08PSQ7BQ/C07EX4ZU15Y)
```